prospector.go

```diff
@@ -104,6 +105,18 @@ func (p *Prospector) scan(path string, output chan *FileEvent, resume *Prospecto
 			continue
 		}
 
+		// If the file is in the list of exclusions, skip it
+		for _, pattern := range exclude_paths {
```

Comment:
Rather than exclude paths as a parameter maybe p.FileConfig here?

It saves the parameter list from growing :)